bindings/cs/Scripts/build.cake

```diff
@@ -0,0 +1,218 @@
+#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+#addin nuget:?package=Cake.FileHelpers&version=3.3.0
+#addin nuget:?package=Cake.Git&version=0.22.0
+#addin nuget:?package=Cake.Incubator&version=5.1.0
+#addin nuget:?package=Cake.Compression&version=0.2.4
+#tool dotnet:?package=GitVersion.Tool&version=5.3.7
+#tool nuget:?package=xunit.runner.console
+
+using System;
+using System.Linq;
+
+var target = Argument("target", "CreatePackage");
+var configuration = Argument("configuration", "Release");
+
+var gitVersion = GitVersion();
+
+var buildDirectory = MakeAbsolute(Directory(GetSettingValue("buildDirectory", "c:\\build")));
+var libmongocryptAllDirectory=buildDirectory.Combine("libmongocrypt-all");
+var downloadedMongocryptDirectory=buildDirectory.Combine("downloadedMongocryptDirectory");
+var localReleaseVersion = "local-0.0.0";
+var releaseVersion = GetSettingValue("releaseVersion", localReleaseVersion);
+var fork = GetSettingValue("fork", "https://github.com/mongodb/libmongocrypt.git");
+var branch = GetSettingValue("branch", "master"); 
+var libmongocryptAllUrl = GetSettingValue("url", "https://mciuploads.s3.amazonaws.com/libmongocrypt/all/master/508e21f4abff9f5519e0357a63a4ad30d2c24692/libmongocrypt-all.tar.gz");
+var csharpBindingsGitTagName = $"csharp-v{releaseVersion}";
+var csharpBindingsDirectory = buildDirectory.Combine(csharpBindingsGitTagName);
+var libmongocryptRelWithDebInfoDirectory = csharpBindingsDirectory.Combine("cmake-build").Combine("RelWithDebInfo");
+var libmongocryptCsDirectory = csharpBindingsDirectory.Combine("bindings").Combine("cs");
+var libmongocryptSolutionDirectory = libmongocryptCsDirectory.Combine("MongoDB.Libmongocrypt");
+var libmongocryptSolutionFile = libmongocryptSolutionDirectory.CombineWithFilePath("MongoDB.Libmongocrypt.csproj");
+var libmongocryptTestsSolutionDirectory = libmongocryptCsDirectory.Combine("MongoDB.Libmongocrypt.Test");
+var artifactsDirectory = buildDirectory.Combine("artifacts");
+
+Task("Prepare")
+    .Does(() =>
+    {
+        if (DirectoryExists(buildDirectory))
```

Comment:
>EnsureDirectoryExists
>CleanDirectory

the above didn't work for me, because I could not remove a git folder from cloned libmongocrypt repo